frontend/src/bundles/common/components/header/styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 100%;
+    padding: 5px;
+    background-color: var(--header-background);
+    border-bottom: 1px solid var(--header-border);
+}
+
+.avatar {
+    position: relative;
+    right: 15px;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+}
+
+.avatar img {
+    width: 40px;
+    height: 40px;
+    border-radius: 200px;
+}
+
+.status {
+    position: absolute;
+    top: 30px;
+    left: 30px;
+    z-index: 1;
+    width: 10px;
+    height: 10px;
+    border: 2px solid var(--status-border);
+    border-radius: 5px;
+}
+
+.status.online {
+    background-color: var(--status-online);
+}
+
+.status.offline {
+    background-color: var(--status-offline);
+}
```

Comment:
```suggestion
.status {
    position: absolute;
    top: 30px;
    left: 30px;
    z-index: 1;
    width: 10px;
    height: 10px;
    border: 2px solid var(--status-border);
    border-radius: 5px;
    
    &.online {
        background-color: var(--status-online);
    }
    &.offline {
        background-color: var(--status-offline);
    }
}
```

Reply:
@nikita-remeslov Earlier there was eslint rule with max depth 0 for styles. As I understand it is allowed now to have depth 1, 
 is it?

Reply:
Yes, I talked about it on our daily